hw2_1.py

```diff
@@ -0,0 +1,10 @@
+i = int(input("Введите число "))
+
+
+h = (format(i, '#x'))
```

Comment:
Подразумевалось решить через деления на 16, но вы нашли доселе никем не используемый способ. Браво
```suggestion
h = format(i, '#x')
```